fakeserver/routes.go

```diff
@@ -128,6 +128,11 @@ func (s *server) routes() {
 		"/api/v1/split_details/{id}",
 		getV1SplitDetail,
 	)
+
+	s.handleGet(
+		"/api/v3/builds/{b}/split_registry",
+		getV2SplitRegistry,
```

Comment:
<< domain lgtm platform lgtm

take-it-or-leave-it nit - should we rename this to `getV2PlusSplitRegistry` or something?